File: Fibonacci/Java/FibonacciSuccessively.java
public class FibonacciSuccessively {
    public static void main(String[] args) {
        // 計算処理
        long start_time = System.nanoTime();
        int result = func(Integer.parseInt(args[0]));


        // 出力
        System.out.println("第" + args[0] + "項" + result);
        long end_time = System.nanoTime();
        System.out.println(end_time-start_time);
        System.out.println("time = " + (Math.pow(10,-6)*(end_time - start_time)) + " [sec]");
    }
    public static int func(int n) {
        if(n < 2){
            return n;
        }
        else{
            return func(n-2) + func(n-1);
        }
    }
}

File: Fibonacci/Python/FibonacciSuccessively.py
import time
import sys

def func(n):
    if n <2:
            return n
    else:
            return func(n-2) + func(n-1)

argv = int(sys.argv[1])
stratTime = time.time()            
#print('第{0}項 = {1}'.format(argv,func(argv)))
func(argv)
endTime = time.time() #プログラムの終了時刻
runTime = endTime - stratTime #処理時間
print('time = {:.10f}sec'.format(runTime)) #処理時間を表示
